src/pbt/v2/client/databricks_client/permission_cpi.py

```diff
@@ -0,0 +1,7 @@
+class PermissionsApi(object):
+    def __init__(self, client):
```

Comment:
can we add type of client? And in general, typing hints in every class?

Reply:
this is the best practice what databricks api classes follow, that's why didn't added  
![Screenshot 2023-08-07 at 10 56 40 PM](https://github.com/SimpleDataLabsInc/prophecy-build-tool/assets/3254558/053e8e13-dbc2-4a44-897e-53ac8d838ce5)